awx/ui_next/src/screens/Job/JobOutput/shared/OutputToolbar.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import styled from 'styled-components';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { shape, func } from 'prop-types';
+import {
+  DownloadIcon,
+  RocketIcon,
+  TrashAltIcon,
+} from '@patternfly/react-icons';
+import { Badge as PFBadge, Button, Tooltip } from '@patternfly/react-core';
+import VerticalSeparator from '@components/VerticalSeparator';
+import DeleteButton from '@components/DeleteButton';
+import LaunchButton from '@components/LaunchButton';
+
+const BadgeGroup = styled.div`
+  margin-left: 20px;
+  height: 18px;
+  display: inline-flex;
+`;
+
+const Badge = styled(PFBadge)`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-left: 10px;
+  ${props =>
+    props.color
+      ? `
+  background-color: ${props.color}
+  color: white;
+  `
+      : null}
+`;
+
+const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  font-size: 14px;
+`;
+
+function toHHMMSS(s) {
```

Comment:
With an elapsed of `4.6` this returns `00:00:26` when the expected value is `00:00:04`.